models/games_models.js

```diff
@@ -7,27 +7,24 @@ exports.selectCategories = () => {
 
 };
 
-exports.selectReviews = (sort_by , category, sort) => {
+exports.selectReviews = (sort_by , category, order) => {
```

Comment:
As Emily mentioned - you can set default arg values up in the parameters list here so that if they haven't been provided, you can still make sure that your reviews are sorted by created_at in desc order by default which will save a bit of code